apps/atomy-q/API/tests/Feature/IdentityGap7Test.php

```
@@ -577,6 +577,435 @@ public function test_protected_route_allows_wildcard_permission_via_role(): void
+        $response->assertJsonPath('data.email', 'invitee@atomy.test');
+        $response->assertJsonPath('data.name', 'Invitee User');
+        $response->assertJsonPath('data.status', 'pending_activation');
+        $response->assertJsonPath('data.tenant_id', (string) $tenant->id);
+
+        $this->assertDatabaseHas('users', [
+            'tenant_id' => $tenant->id,
+            'email' => 'invitee@atomy.test',
+            'name' => 'Invitee User',
+            'status' => 'pending_activation',
+        ]);
+    }
```

PHPMD (2.15.0)

[error] The method test_users_invite_creates_persisted_pending_record is not named in camelCase. (undefined)

(CamelCaseMethodName)